internal/runmode/runmode.go

```diff
@@ -0,0 +1,105 @@
+package runmode
+
+import (
+	"fmt"
+
+
+	"time"
+
+	gh "github.com/google/go-github/github"
```

Comment:
Note: gh -> github and own github packages can result in misleading the developer/reader. I think your own github package is not "exactly" github, so that might be subject to be changed.

Reply:
Indeed, the google github package should be internalised by our github wrapper package, or maybe the wrapper should be a generic one (e.g `issuetracker`).

It would be a big refactor however, which I'd prefer doing in a separate PR.